tests/testDipoleFitter.py

```diff
@@ -220,12 +238,14 @@ def testDipoleTask(self):
         return result
 
     def testDipoleEdge(self):
-        """Test the dipole fitting singleFramePlugin. Test that the dipoles
-        which are too close to the edge raise the correct exception.
+        """Test the edge case for dipole fitting singleFramePlugin.
+
+        Test that the dipoles which are too close to the edge are
+        indicated as so in the catalog.
```

Comment:
In this case, no dipoles will be found. But you are right, given the input coords I set, it only detected one diaSource (should be two). I will update the coordinates so both diaSources are detected but that they are still too close to the edge to be measured.